nifi-registry/nifi-registry-core/nifi-registry-flow-diff/src/main/java/org/apache/nifi/registry/flow/diff/StandardFlowDifference.java

```diff
@@ -89,8 +89,8 @@ public String toString() {
 
     @Override
     public int hashCode() {
-        return 31 + 17 * (componentA == null ? 0 : componentA.getIdentifier().hashCode()) +
-            17 * (componentB == null ? 0 : componentB.getIdentifier().hashCode()) +
+        return 31 + 17 * (componentA == null ? 0 : Objects.hash(componentA.getIdentifier())) +
+            17 * (componentB == null ? 0 : Objects.hash(componentB.getIdentifier())) +
```

Comment:
This method should use Objects.`hashCode(Object o)` instead of Objects.`hash(Object... values)`. The `Objects.hash()` method delegates to `Arrays.hashCode()`, but all that is necessary is to use `Objects.hashCode()` to provide null-safe handling of `identifier`.
```suggestion
        return 31 + 17 * (componentA == null ? 0 : Objects.hashCode(componentA.getIdentifier())) +
            17 * (componentB == null ? 0 : Objects.hashCode(componentB.getIdentifier())) +
```

Reply:
Hey David, thanks for the catch, will make those changes